coreos-with-fleet-multivm/azuredeploy.json

```diff
@@ -0,0 +1,230 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "West US",
+            "metadata": { 
+                 "description": "Location of resources" 
+            } 
+        },
+        "newStorageAccountName": {
+            "type": "string",
+            "defaultValue": "storageaccount",
+            "metadata": { 
+                 "description": "Name of the storage account for VM OS Disks"
+            } 
+        },
+        "vmNamePrefix": {
+            "type": "string",
+            "defaultValue": "core",
+            "metadata": { 
+                 "description": "Prefix for the names of the VMs"
+            } 
+        },
+        "virtualNetworkName": {
+            "type": "string",
+            "defaultValue": "myVNET",
+            "metadata": { 
+                 "description": "Name of the VNET hosting the VMs"
+            } 
+        },
+        "imageSku": {
+            "type": "string",
+            "defaultValue": "Stable",
+            "allowedValues": [ 
+                "Stable", 
+                "Beta", 
+                "Alpha" 
+            ], 
+            "metadata": { 
+                 "description": "Specifies the CoreOS image SKU"
+            } 
+        },
+        "imageVersion": {
+            "type": "string",
+            "defaultValue": "633.1.0",
+            "metadata": { 
+                 "description": "Specifies the CoreOS image version"
+            } 
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_A1",
```

Comment:
I did consider doing that but the problem is that there are different values depending on whether the SKU is Alpha, Beta, or Stable.